### DISCUSSION TITLE
dotnet cli

### DISCUSSION THREAD

**Discussion by 563256676** [dotnet/runtime#61645] [Q&A]
When I run> dotnet remove package Microsoft.NETCore.App
An error occurred. How can I make it run successfully? I installed this package from NuGet.

Information: Delete the PackageReference of the package "Microsoft.NETCore.App" from the project "C:\Users\appeon\source\repos\ConsoleApp59\ConsoleApp59\ConsoleApp59.csproj".
Error: An error occurred while deleting the package "Microsoft.NETCore.App". Cannot edit items in the import file-
Error: Import the project "PackageReference" of "Microsoft.NETCore.App" in the file "C:\Program Files\dotnet\sdk\6.0.100-rc.2.21505.57\Sdks\Microsoft.NET.Sdk\targets\Microsoft" . NET.Sdk.DefaultItems.props'

**Comment by huoyaoyuan** [comment]
The metapackage `Microsoft.NETCore.App` is imported by the SDK, and you can't modify it.

> I installed this package from NuGet.

This should be error or no-op. You shouldn't have really completed this step.

**Reply by deeprobin** [reply]
I think we should forbid this (maybe a --allow-dangerous option to allow this)
